qkit/drivers/Keysight_VNA_E5080B.py

```diff
@@ -870,7 +871,7 @@ def post_measurement(self):
         """
         Bring the VNA back to a mode where it can be easily used by the operator.
         """
-        self.set_sweep_mode("hold")
```

Comment:
Isn't hold better to not accidently excite/heat your sample? Also, one can directly see in the lab if a measurement is running or not

Reply:
I guess that's a matter of personal taste. But we should have it equal on all devices/drivers. 

The primary intention of this command was that you don't have to do anything to be able to measure at the device directly after a measurement command. So it has to be running. But I see your point.